app/presenters/publishing_api_presenters/item.rb

```diff
@@ -0,0 +1,66 @@
+module PublishingApiPresenters
+  class Item
+    extend Forwardable
+
+    def_delegators :item, :base_path, :content_id, :title, :description, :need_ids, :public_updated_at
+    attr_accessor :update_type
+
+    def initialize(item, update_type: nil)
+      self.item = item
```

Comment:
Is there a reason for using the accessors here, rather than assigning to an ivar as elsewhere?